packages/dds/shared-summary-block/src/sharedSummaryBlock.ts

```diff
@@ -97,7 +98,7 @@ export class SharedSummaryBlock extends SharedObject implements ISharedSummaryBl
     /**
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.snapshot}
      */
-    public snapshot(): ITree {
+    protected snapshotCore(serializer: IFluidSerializer): ITree {
```

Comment:
Places like that (that take serializer but do not use it) are a suspect. Are we certain data stored in this DDS can't have handles? Do we enforce it somehow (through strong typing of values that goes in)?
If not, it feels like a candidate to open an issue against and figure out what's the plan

Reply:
SharedSummaryBlock expects the data to be of type "Jsonable" which cannot be a handle (from what I can tell). The other DDS that don't use serializer also do not support handles.
But its worth investigating. I have created task #4329 to investigate this.